lib/src/material/material_controls.dart

```diff
@@ -366,33 +369,56 @@ class _MaterialControlsState extends State<MaterialControls>
     final bool isFinished = _latestValue.position >= _latestValue.duration;
     final bool showPlayButton =
         widget.showPlayButton && !_dragging && !notifier.hideStuff;
-
-    return GestureDetector(
-      onTap: () {
-        if (_latestValue.isPlaying) {
-          if (_displayTapped) {
-            setState(() {
-              notifier.hideStuff = true;
-            });
-          } else {
-            _cancelAndRestartTimer();
-          }
-        } else {
-          _playPause();
-
-          setState(() {
-            notifier.hideStuff = true;
-          });
-        }
-      },
-      child: CenterPlayButton(
-        backgroundColor: Colors.black54,
-        iconColor: Colors.white,
-        isFinished: isFinished,
-        isPlaying: controller.value.isPlaying,
-        show: showPlayButton,
-        onPressed: _playPause,
-      ),
+    final bool showSeekButton =
+        widget.showSeekButton && !_dragging && !notifier.hideStuff;
+
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        SeekRewindButton(
+          backgroundColor: Colors.black54,
+          iconColor: Colors.white,
+          show: showSeekButton,
+          onPressed: _seekRewind,
+          onDoublePressed: _seekRewind,
+          icon: Icons.fast_rewind,
+        ),
+        GestureDetector(
+          onTap: () {
+            if (_latestValue.isPlaying) {
+              if (_displayTapped) {
+                setState(() {
+                  notifier.hideStuff = true;
+                });
+              } else {
+                _cancelAndRestartTimer();
+              }
+            } else {
+              _playPause();
+
+              setState(() {
+                notifier.hideStuff = true;
+              });
+            }
+          },
+          child: CenterPlayButton(
+            backgroundColor: Colors.black54,
+            iconColor: Colors.white,
+            isFinished: isFinished,
+            isPlaying: controller.value.isPlaying,
+            show: showPlayButton,
+            onPressed: _playPause,
+          ),
+        ),
+        SeekRewindButton(
+          backgroundColor: Colors.black54,
+          iconColor: Colors.white,
+          show: showSeekButton,
+          onPressed: _seekForward,
+          onDoublePressed: _seekForward,
+          icon: Icons.fast_forward,
+        ),
+      ],
```

Comment:
Duplicated code from the other file: `lib/src/cupertino/cupertino_controls.dart`